packages/@aws-cdk/aws-eks/README.md

```diff
@@ -45,6 +47,45 @@ cluster.addResource('mypod', {
 });
 ```
 
+In order to interact with your cluster through `kubectl`, you can use the `aws
+eks update-kubeconfig` [AWS CLI](https://aws.amazon.com/cli/) command in order
+to configure your local kubeconfig.
+
+The EKS module will define a CloudFormation output in your stack which contains
+the command to run. For example:
+
+```
+Outputs:
+ClusterConfigCommand43AAE40F = aws eks update-kubeconfig --name cluster-xxxxx --role-arn arn:aws:iam::112233445566:role/yyyyy
+```
+
+> The IAM role specified in this command is called the "**masters role**". This is
+> an IAM role that is associated with the `system:masters` [RBAC] group and has
+> super-user to the cluster. You can specify this role using the `mastersRole`
+> option, or otherwise a role will be automatically created for you.
+
+Execute the `aws eks update-kubeconfig ...` command in your terminal to create a
+local kubeconfig:
+
+```console
+$ aws eks update-kubeconfig --name cluster-xxxxx --role-arn arn:aws:iam::112233445566:role/yyyyy
+Added new context arn:aws:eks:rrrrr:112233445566:cluster/cluster-xxxxx to /home/boom/.kube/config
+```
+
+And now you can simply use `kubectl`:
+
+```console
+$ kubectl get all -n kube-system
+NAME                           READY   STATUS    RESTARTS   AGE
+pod/aws-node-fpmwv             1/1     Running   0          21m
+pod/aws-node-m9htf             1/1     Running   0          21m
+pod/coredns-5cb4fb54c7-q222j   1/1     Running   0          23m
+pod/coredns-5cb4fb54c7-v9nxx   1/1     Running   0          23m
+...
+```
+
+[RBAC]: https://kubernetes.io/docs/reference/access-authn-authz/rbac/
```

Comment:
Why here? You can inline this inside the appropriate block quote

Reply:
Done